torch/autograd/_functions/basic_ops.py

```diff
@@ -169,6 +169,14 @@ def backward(ctx, grad_output):
 
 @traceable
 class MulConstant(InplaceFunction):
+    @staticmethod
+    def symbolic(g, a, b, inplace=False):
+        assert not inplace, "Exporting inplace MulConstant is not supported yet"
+        if isinstance(a, torch._C.Node):
```

Comment:
Can we write a helper function here similar to `sort_args`; most importantly to avoid duplicating the very ugly `isinstance` check.